.github/workflows/acceptance.yml

```diff
@@ -212,17 +225,10 @@ jobs:
           save-only: true
           snapshot-path: .artifacts/visual-snapshots
 
-      # We use the below to determine the flag to use in codecov
-      - name: Check for python file changes
-        uses: getsentry/paths-filter@v2
-        id: changes
-        with:
-          token: ${{ github.token }}
-          filters: .github/file-filters.yml
-
       - name: Handle artifacts
         uses: ./.github/actions/artifacts
-        if: ${{ steps.changes.outputs.backend == 'true' }}
+        # In order to determine the flag to use in codecov
```

Comment:
This doesn't determine the flag, it just completely skips uploading anything if it's not backend.